builder/solver_monitor.go

```diff
@@ -264,6 +287,60 @@ func (sm *solverMonitor) printHeader(vm *vertexMonitor) {
 	vm.printHeader(!seen || sm.verbose)
 }
 
+func (sm *solverMonitor) recordTiming(targetStr, targetBrackets, salt string, vertex *client.Vertex) {
+	if vertex.Started == nil || vertex.Completed == nil {
+		return
+	}
+	dur := vertex.Completed.Sub(*vertex.Started)
+	if dur == 0 {
+		return
+	}
+	key := timingKey{
+		targetStr:      targetStr,
+		targetBrackets: targetBrackets,
+		salt:           salt,
+	}
+	sm.timingTable[key] = sm.timingTable[key] + dur
```

Comment:
not that it really matters, but is it not possible to do this instead?
```suggestion
	sm.timingTable[key] += dur
```

Reply:
:+1: